.ci/install_runtime.sh

```diff
@@ -46,6 +48,7 @@ OPENSHIFT_CI="${OPENSHIFT_CI:-false}"
 
 runtime_config_path="${SYSCONFDIR}/kata-containers/configuration.toml"
 runtime_src_path="${GOPATH}/src/${KATACONTAINERS_REPO}/src/runtime"
+agent_ctl_path="${GOPATH}/src/${KATACONTAINERS_REPO}/src/tools/agent-ctl"
```

Comment:
use `katacontainers_repo_dir` variable. The `runtime_src_path` and other variables on this file are doing wrong.